composer.json

```diff
@@ -80,14 +79,12 @@
     "scripts": {
         "website:copy-changelog": "@php -r \"copy('CHANGELOG.md', 'docs/changelog.md');\"",
         "website:copy-logo": "@php -r \"(is_dir('docs/.vuepress/public') || mkdir('docs/.vuepress/public')) && copy('art/logo_mixed.gif', 'docs/.vuepress/public/logo.gif') && copy('art/heart.svg', 'docs/.vuepress/public/heart.svg') && copy('art/heart.png', 'docs/.vuepress/public/heart.png');\"",
-        "ecs:test": "ecs check src --ansi --config vendor/symplify/easy-coding-standard/config/set/clean-code.php",
```

Comment:
Maybe we should configure our .php_cs file with fixer listed here ? https://github.com/symplify/easy-coding-standard/blob/main/config/set/clean-code.php

Reply:
Be careful also on github workflow that run this task here https://github.com/nunomaduro/phpinsights/blob/master/.github/workflows/format.yaml#L45-L46

Reply:
anyway, handle by #494 :wink:

Reply:
Great! Thanks.